share/ert/site-config

```diff
@@ -1,5 +1,6 @@
 -- This is a very basic site config file, mainly used for testing.
 
+WORKFLOW_JOB_DIRECTORY workflows/jobs/shell
```

Comment:
The equivalent of this line also needs to be added here ([link](https://github.com/equinor/ert/blob/8f2e7181123c53555520b1d3359b15c893b3b5e3/ert_shared/hook_implementations/jobs.py#L103)) for the plugin system to install them 🙂

Reply:
Will you make an issue for doing that as well? After doing that this looks good to me.

Reply:
Already made a PR Equinor/ert#1029.